locales/volto.pot

```diff
@@ -737,6 +738,11 @@ msgstr ""
 msgid "Content rules from parent folders"
 msgstr ""
 
+#: components/manage/LinksToItem/LinksToItem
+# defaultMessage: Content that link to this item
+msgid "Content that link to this item"
+msgstr ""
+
```

Comment:
I think this message is redundant to the heading above it in the image, `Links to "Supervisor Adélaïde Pickavance"`, and I find it confusing. I would remove it.